src/zorak/cogs/admin/verification_on_verified.py

```diff
@@ -66,7 +120,7 @@ async def add_verify_button(self, ctx):
         """Adds the button"""
         button_message = ("Welcome to Practical Python! We’re thrilled to have you here to learn "
                           "and collaborate with fellow Python enthusiasts. Before diving in, "
-                          "please click the 'Verify' button to confirm your identity and gain "
+                          "please click the **GREEN** 'Verify' button to confirm your identity and gain "
```

Comment:
Should also add a warning, that if the user clicks the WRONG button, they will be kicked.